sdk/core/Azure.Core/src/Pipeline/HttpHeader.cs

```diff
@@ -61,10 +61,12 @@ public static class Common
         {
             static readonly string s_applicationJson = "application/json";
             static readonly string s_applicationOctetStream = "application/octet-stream";
+            static readonly string s_applicationForm = "application/x-www-form-urlencoded";
```

Comment:
It wopuld be better if the name of the string and the header were similar.

Reply:
Agreed, fixed.